packages/web3-bzz/package.json

```diff
@@ -9,17 +9,27 @@
     },
     "types": "types/index.d.ts",
     "scripts": {
-        "dtslint": "dtslint types --onlyTestTsNext"
+        "dtslint": "dtslint types --onlyTestTsNext",
+        "build:dev": "../../scripts/build/build.dev.sh",
+        "build:cjs": "../../scripts/build/build.cjs.sh",
+        "build:esm": "../../scripts/build/build.esm.sh",
+        "build:minified": "../../scripts/build/build.min.sh",
+        "watch": "../../scripts/build/watch.dev.sh"
     },
-    "main": "src/index.js",
+    "main": "dist/web3-bzz.cjs.js",
+    "module": "dist/web3-bzz.esm.js",
```

Comment:
Yes, the ``build:dev`` which does get executed does add the ``istanbul`` instrumentation for later creating a coverage report. Do you propose another workflow on ``npm i``?

Reply:
Maybe we should use a different output file for istanbul instrumented files. Is this feasible?

Reply:
Hmm.. yes this would be a possible solution. 🤔

Reply:
I think we can keep this for now and improve it later because the mentioned re-structuring of the package structure will simplify the project setup and solve the instrumentation issue you have here.

Reply:
I agree that this can be addressed in the future. Can we add a minor comment about this so new contributors don't freak out? 😂